### DISCUSSION TITLE
Best approach to updating static site generator for GH Pages

### DISCUSSION THREAD

**Discussion by dstack4273** [community/community#27736] [Other Feature Feedback, Questions, & Ideas]
Hello, 

I've decided I want to actually do some blogging again, but it's been ages since I did anything to maintain or otherwise handle what is already running for my personal GH page. I'd like to change out the static site generator and try something else, but I'm not really sure what the best way to approach that might be. I've poked around the web and haven't really found a satisfying answer. 

Should I just delete the repo and start from scratch? Is there some other approach that makes sense? Has anyone done this before 😆 ?

Thanks for your help and thoughts!

**Comment by airtower-luna** [comment]
It really depends on what you want. :wink: 

Usually when I want to make a major change to any of my projects I make a branch, experiment, merge it if I'm happy. If you want to see an intermediate state published without overwriting your old page, you could push that branch to a new (temporary) repository for that, or (if you don't mind taking the old version offline) change the publishing branch in your repository settings.

Of course if you just want to completely scrap your old site and start fresh deleting is a valid choice.

Either way this update to how pages are published may be interesting for you: https://github.blog/changelog/2022-07-27-github-pages-custom-github-actions-workflows-beta/

**Comment by jfullstackdev** [comment]
to add, you can simply leave your old page and start creating one, because there is no limit

you can even create an Org so you will have another page that is not the same as your personal account, for example me, I have this Org with GitHub Pages, and it's intentional, I'll be putting contents there separate from my personal account

https://github.com/devz-throwable

the page, as we know the pattern is, 

<https://devz-throwable.github.io/>

![image](https://user-images.githubusercontent.com/47092464/182031578-2af146c5-35ee-4d97-b8f8-3ff497e9ecdc.png)
